hazelcast/src/main/java/com/hazelcast/internal/partition/impl/PartitionStateGeneratorImpl.java

```diff
@@ -39,16 +39,21 @@
 
 final class PartitionStateGeneratorImpl implements PartitionStateGenerator {
 
-    private static final ILogger LOGGER = Logger.getLogger(PartitionStateGenerator.class);
-
     private static final int DEFAULT_RETRY_MULTIPLIER = 10;
 
     private static final float RANGE_CHECK_RATIO = 1.1f;
+
     private static final int MAX_RETRY_COUNT = 3;
     private static final int AGGRESSIVE_RETRY_THRESHOLD = 1;
     private static final int AGGRESSIVE_INDEX_THRESHOLD = 3;
     private static final int MIN_AVG_OWNER_DIFF = 3;
 
+    private final ILogger logger;
+
+    public PartitionStateGeneratorImpl(LoggerFactory loggerFactory) {
```

Comment:
is logging service unavailable yet? as logger factory will not the the fancy stuff with member identification, etc